index.html

```diff
@@ -2566,22 +2566,21 @@ <h3>Inverse Triple Constraints</h3>
   }
```

Comment:
"For inverse triple constraints, we have those as an integrated example
because ShEx itself allows you to get mixed results. We want to convey
the idea that you can have a block of data that gets mixed results."

A try-it button would make this clear.